cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1341,6 +1341,32 @@
     </description>
   </property>
 
+  <!-- Operational Stats Configuration -->
+  <property>
+    <name>operational.stats.extensions.dir</name>
+    <value>/opt/cdap/master/ext/operations</value>
+    <description>
+      Semicolon-separated list of local directories that are scanned for
+      operational stats extensions
+    </description>
+  </property>
+
+  <property>
+    <name>operational.stats.collection.threads</name>
```

Comment:
Why we need that many threads. From the logic, it seems like only one thread is needed (you use `scheduleWithFixedDelay`, hence it won't have overlapping run).